profile_inference.py

```
@@ -695,6 +713,605 @@ def run_benchmark_mode(dit_handler, llm_handler, args, timer: PreciseTimer):
+
+    dit_handler = None
+    llm_handler = None
+
+    try:
+        print(f"\n  Initializing DiT handler... ({_get_vram_info_str()})")
+        dit_handler = AceStepHandler()
+
+        # Determine flash attention availability
+        use_flash_attention = False
+        try:
+            import flash_attn  # noqa: F401
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)